operations/research-and-development/engineering/team/README.md

```diff
@@ -61,3 +40,32 @@ Below is how SET prioritizes what is worked on.
    * Pick up any bugs in triage without an obvious owning feature team.
```

Comment:
I'd remove the ownership part, it might be an important bug that the team supposed to own can't handle due to the ammount of work.
```suggestion
   * Pick up any bugs in triage.
```